src/main/resources/net/snowflake/ingest/ingest_error_messages.properties

```diff
@@ -34,3 +34,4 @@
 0029=Data type not supported: {0}
 0030=The given row cannot be converted to the internal format due to invalid value: {0}
 0031=The given row exceeds the maximum allowed row size {0}
+0035=The given batch of rows passed to insertRows() exceeded the maximum allowed batch size of {0} bytes. Consider splitting the batch into multiple smaller batches before passing them to insertRows(). We recommend batch size does not exceed {1} bytes.
```

Comment:
nit: can we make the message more strict since we are throwing an error on this?
"We recommend batch size does not exceed {1} bytes." -> "The batch size cannot exceed {1} bytes."

Reply:
See my comment below.